W-13735979-Review-Debug-a-Mule-app-CP (#66)
mulesoft/docs-code-builder: modules/ROOT/pages/debug-add-breakpoints.adoc

. In Anypoint Code Builder, on the Explorer view, open your `interface.xml`.
. Select the *Run and Debug* icon in the activity bar:
+
image::run-and-debug.png["Run and debug icon highlighted in the activity bar"]
. Click next to the line in your `<flow-ref>` component to add a breakpoint on that component.
// TODO: There may be a step missing in the implement-api-interface-logic.adoc file. The `<flow-ref>` is not available for me. I have a <ee:transform doc:name="Transform Message">. Can we add here something like "<flow-ref name="AnotherFlow" />"?
+
image::add-breakpoint.png["Breakpoint highlighted next to the line number"]
+

> NOTE: You can also use `F9` to add a breakpoint to the line you are standing on.

Reviewer: ```suggestion
NOTE: You can also use `F9` to add a breakpoint to the current line.
```

+

> . On your `implementation.xml` file, add two more breakpoints for the `<db:select>` and the `<set-variable>` components. Your breakpoints show up in the *Breakpoints* panel:

Reviewer: ```suggestion
. In your `implementation.xml` file, add two more breakpoints for the `<db:select>` and the `<set-variable>` components. Your breakpoints appear in the *Breakpoints* panel:
```

// TODO: Panel name is always in capital letters both in web IDE and in desktop IDE.

image::breakpoints-panel.png["Breakpoints added highlighted"]

== Next Step

* xref:run-a-debug-session.adoc[Run a Debug Session]. +

> Run your first debug session and inspect the message and variables at different points during your application execution.

Reviewer: ```suggestion
Run your first debug session and inspect the message and variables at different points during application execution.
```
